packages/core/src/ui/components/Account/EnableAccountPasswordPrompt/EnableAccountPasswordPrompt.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { Box, Button, Flex, PasswordBox, Text } from '@lace/ui';
+import { Drawer, DrawerNavigation } from '@lace/common';
+
+interface Props {
+  open: boolean;
+  password: string;
+  wasPasswordIncorrect: boolean;
+  translations: {
+    title: string;
+    description: string;
+    passwordPlaceholder: string;
+    wrongPassword: string;
+    cancel: string;
+    confirm: string;
+  };
+  onCancel: () => void;
+  onConfirm: (password: string) => void;
+  isPopup: boolean;
+}
+
+export const EnableAccountPasswordPrompt = ({
+  open,
+  password = '',
+  wasPasswordIncorrect,
+  isPopup,
+  onConfirm,
+  onCancel,
+  translations
+}: Props): JSX.Element => {
+  const [currentPassword, setCurrentPassword] = useState(password);
+
+  return (
+    <Drawer
+      zIndex={1100}
+      open={open}
+      navigation={<DrawerNavigation onArrowIconClick={onCancel} />}
+      onClose={onCancel}
```

Comment:
I'm not sure whether this would make any difference for time in memory since the component is probably dismounted and useState variables dereferenced? but it might
```suggestion
      onClose={() => {
        onCancel();
        setCurrentPassword('');
      }}
```

Reply:
fixed in b80c193c7839186c6059b7703a5764cf39f37085